### DISCUSSION TITLE
Second car issues

### DISCUSSION THREAD

**Discussion by slavag** [teslamate-org/teslamate#2880] [Q&A] [answered]
Hi,
I added second car to my account and TeslaMate is pulling data fine for that car as well,
But only part of data for example I don't see any consumption :
<img width="1308" alt="image" src="https://user-images.githubusercontent.com/511540/188370650-66d24da8-5991-4d94-ae9e-5b29005e160a.png">

Any idea what can be an issue here ?

Thanks

**Comment by cwanja** [comment, marked as answer]
Check out the FAQs. https://docs.teslamate.org/docs/faq#why-are-no-consumption-values-displayed-in-grafana
